hugo/content/docs/learn/getting-started.md

```diff
@@ -0,0 +1,180 @@
+---
+title: 'Getting Started'
+---
+
+**Welcome to the first step of your Sprotty journey!** In this section, we'll set up everything you need to start building interactive diagrams.
+
+![Project Setup](/images/project-setup.png)
```

Comment:
this png is not found. which one should it be?

Reply:
This shouldn't be here